.github/release.yml

```diff
@@ -0,0 +1,16 @@
+changelog:
```

Comment:
How does GitHub know to use this file? Is it standard procedure? Its the first time I see it

Reply:
if you see release notes. you can see the details from changelog which we need to manually generate. But with this it will be automatic. I am thinking in future we can incorporate release notes into the documentation. 
I was following the template from pymapdl. 

@germa89  let me know if my understanding is correct.

Reply:
why me? XDD